src/schema/users.ts

```diff
@@ -850,14 +850,61 @@ export const resolvers: IResolvers<any, Context> = {
         .orderBy('date')
         .getRawMany();
     },
-    userStreak: (_, __, ctx: Context, info): Promise<GQLUserStreak> =>
-      graphorm.queryOneOrFail<GQLUserStreak>(ctx, info, (builder) => ({
-        queryBuilder: builder.queryBuilder.where(
-          `"${builder.alias}"."userId" = :id`,
-          { id: ctx.userId },
-        ),
-        ...builder,
-      })),
+    userStreak: async (_, __, ctx: Context, info): Promise<GQLUserStreak> => {
+      const streak = await graphorm.queryOneOrFail<GQLUserStreak>(
+        ctx,
+        info,
+        (builder) => ({
+          queryBuilder: builder.queryBuilder.where(
+            `"${builder.alias}"."userId" = :id`,
+            { id: ctx.userId },
+          ),
+          ...builder,
+        }),
+      );
```

Comment:
The `lastViewAt` has timezone as part of its data. We can probably just turn it to UTC on return so the value would match with `new Date()`.

Reply:
[See my comment on slack](https://dailydotdev.slack.com/archives/C05TQL6HWTH/p1706629059323989?thread_ts=1706625076.569029&cid=C05TQL6HWTH), it's not the way we thought :) 

On top of that, the RFC mentions that we want to do all of the operations on these dates in user's timezone. I think clearing should take that into account as well, otherwise we would inconsistently clear the streaks.

Reply:
Should now take into account user's timezone.